gyp_xwalk

```diff
@@ -351,6 +351,9 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  if sys.platform == 'win32':
```

Comment:
This discussion is getting too lengthy for a PR comment, especially after the PR has already been merged. I've created XWALK-6207 to track this, let's continue there.